Homework10_2.py

```diff
@@ -0,0 +1,45 @@
+"""Вернуть число"""
+
+
+def check_result(func):
+    """
+    Декоратор, проверяющий,
+    является ли результат функции числом
+
+    Args:
+        Результат выполнения функцией операции над аргументами
+
+    Returns:
+        Возбуждает исключение ValueError
+        в случае, если результат функции не является числом
+        Возвращает функцию, если результат является числом
+    """
+    def wrapper(arg1, arg2):
```

Comment:
better to expect `*args` different amount of arguments

Reply:
Done